src/dlgTriggerEditor.cpp

```diff
@@ -196,7 +196,7 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
              this,
              SLOT(slot_updateStatusBar(QString))
          );
-    simplifyEdbeeStatusBarRegex = new QRegularExpression(R"(^(.+?) \|)");
+    simplifyEdbeeStatusBarRegex = new QRegularExpression(R"(^(?:\[\*\] )?(.+?) \|)");
```

Comment:
:-1: I think this will need/ought to be reverted as I suspect fixing the undo/redo history issue will also cure this - so it actually works as intended - i.e. `[*]` shows up __only__ when the content has been __changed__ from what was initially painted into the widget...